sdks/python/apache_beam/utils/retry.py

```diff
@@ -42,9 +43,11 @@
 try:
   from apitools.base.py.exceptions import HttpError
   from google.api_core.exceptions import GoogleAPICallError
+  from google.api_core import exceptions
```

Comment:
to make code more readable, would it be better to use import alias to clearly distinguish between `import requests.exceptions`

Reply:
Removed as this is no longer needed